metadata-transformer/src/main/java/no/unit/nva/metadata/NoLowercaseDCPrefixInMetadataRule.java

```diff
@@ -0,0 +1,30 @@
+package no.unit.nva.metadata;
+
+import org.apache.any23.validator.*;
+import org.w3c.dom.Node;
+
+import java.util.List;
+
+public class NoLowercaseDCPrefixInMetadataRule implements Rule {
+
+    @Override
+    public String getHRName() {
+        return "no-lowercase-dc-prefix-in-metadata-rule";
+    }
+
+    @Override
+    public boolean applyOn(DOMDocument document, RuleContext<?> context,
+                           ValidationReportBuilder validationReportBuilder) {
+        List<Node> metas = document.getNodes("/HTML/HEAD/META");
+        for (Node meta : metas) {
+            Node nameNode = meta.getAttributes().getNamedItem("name");
+            if (nameNode != null && nameNode.getTextContent().startsWith("dc.")) {
+                validationReportBuilder.reportIssue(ValidationReport.IssueLevel.ERROR,
```

Comment:
is there a test for the validation report?

Reply:
> Since many tests are merged together to a single parametrized test, I would suggest that we name each string instance with the case that is being tested. Otherwise in a later refactoring it will be impossible to understand what is the role of each string instance.

That is the intent of this PR.